content/en/docs/releases/supported-releases/index.md

```diff
@@ -54,6 +54,7 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | Version         | Currently Supported  | Release Date      | End of Life            | Supported Kubernetes Versions | Tested, but not supported |
 |-----------------|----------------------|-------------------|------------------------|-------------------------------|---------------------------|
 | master          | No, development only |                   |                        |                               |                           |
+| 1.12            | Yes                  | November 18, 2021 | ~May 2022 (Expected)   | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
```

Comment:
We should probably extend the EOL dates out 6 weeks for 1.10 and later, at least I believe these dates seem to be base on the 3 months, not 3 months plus 6 weeks.

Reply:
I believe 1.10 will be late December, so just adding a month might be acceptable. @therealmitchconnors Did we decide on a 6 weeks across the board, or is it still up for debate?

Reply:
I changed the dates to be a month later but happy to change back if we need